scripts/train/yamls/finetune/mpt-7b_dolly_sft.yaml

```diff
@@ -0,0 +1,125 @@
+max_seq_len: 2048
+global_seed: 17
+
+# Run Name
+run_name: # If left blank, will be read from env var $COMPOSER_RUN_NAME
+
+model:
+  name: hf_causal_lm
+  device: cpu
```

Comment:
```suggestion
```
`device` isn't actually a config option, despite what all our YAMLs say. OOPS!
It's supposed to be `init_device`, but we can just remove the option because it needs to be `cpu` (which it defaults to) when you have `pretrained: true`, which all of these obviously will.